piccolo_api/shared/auth/user.py

```diff
@@ -24,7 +24,7 @@ def auth_table(self) -> t.Type[Table]:
 
     @property
     def user_id(self) -> int:
-        return t.cast(int, self.user.id)
+        return t.cast(int, self.user._meta.columns[0])
```

Comment:
And here too `table._meta.primary_key`.

Reply:
@aminalaee Thanks. I'll fix that.

Reply:
@sinisaos this is great BTW. Now all of them have mypy.

Reply:
@aminalaee Thank you for your suggestion because I completely forgot about `table._meta.primary_key`